### DISCUSSION TITLE
deltalake v0.4.1

### DISCUSSION THREAD

**Discussion by MykhailoHevak** [delta-io/delta-rs#820] [Q&A] [answered]
The last version of the rust-library which is in the registry (crates.io) is 0.4.0. And the last release was about 1 year ago. But ruby code which is used in the last python-library has been changed a lot! So, the question is how I can get the last version of the ruby-library which is used in the last python-library?

**Comment by roeap** [comment, marked as answer]
I am not sure about some of the specific of this question :). Yes, the last rust release is a long time ago, and we have been making quite a few python releases since then. However the ruby bindings are not quite related to any of this :).

I.e. The way to get the latest rust version is to specify the git repo in your Cargo.toml. The python bindings are releases regularly, but if you need a more bleeding edge version, they can be build from master - have a look at `python/Makefile`.

As for ruby, right now no one is really working on the ruby bindings and they are extremely experimental. To be honest, unless someone takes up responsibility to drive this topic, I don't see much happening with the ruby bindings any time soon.

**Reply by MykhailoHevak** [reply]
Thanks! I'm not familiar with rust's ecosystem :) So, didn't think about getting rust-library directly from the repository.
